inst/extdata/zerocurve.csv

```diff
@@ -1,28 +1,28 @@
 start,end,zeros,dfs

```

Comment:
why has this file changed?

Reply:
Not totally sure, but we had to change the change the data in the zero curve to fix the bug in the discount factor interpolation.

Reply:
You need to explain why this changed (didn't it work previously?).

Reply:
What happened here is: The data in zerocurve.csv was wrong (wrong pillar points) which caused the zero interpolation of build_zero_curve() to return wrong results. So we have fixed the bug in the commit  "sefhamada:fixing-the-zero-curve" by changing the data in the file. This was done in the master branch. So I have changed the zerocurve.csv data in this fork to match the data in the master.